React/Exercise/src/pages/Ex3.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import Button from '../components/Button'
+import Header from '../components/Header'
+
+const Custom_Component = () => {
```

Comment:
component should not be arrow function

Reply:
Hey @Nkaka23dev,  I don't get this well. Is there any reason for me to not use the arrow function will it affect my code base??? please lemme know on how it can affect my code because I'm a bit confused now